bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/thing/Rollershutter.xml

```diff
@@ -46,12 +46,13 @@
 			<parameter name="receivingEEPId" type="text" multiple="true" required="true">
 				<label>EEP for Receiving States</label>
 				<options>
-					<option value="A5_3F_7F_EltakoFSB">Eltako FSB14/61/71</option>
+					<option value="A5_3F_7F_EltakoFSB">Eltako FSB14/61/71, FJ62</option>
 					<option value="A5_3F_7F_EltakoFRM">Eltako FRM60</option>
 					<option value="D2_05_00">D2-05-00 Rollershutter</option>
 					<option value="D2_05_00_NODON">NODON SIN-2-RS-01 (EEP: D2-05-00)</option>
 					<option value="A5_11_03">A5-11-03 Rollershutter status</option>
 					<option value="F6_00_00">PTM200 Rollershutter status</option>
+					<option value="F6_00_00_EltakoFJ62">Eltako FJ62 status</option>
```

Comment:
@spledl Did you tried to use the already implemented F6_00_00 EEP (PTM200 Rollershutter status)? The implementation of this EEP looks identical to yours. Maybe it is enough to add a hint to FJ62 to the description of the F6_00_00 EEP?

Reply:
@fruggy83 
The code looks very similar, but in the PTM200 Implementation there are no code for handling the "Start up (0x01)" and "Start down (0x02)" status information of the Eltako FJ62 rollershutter.

see also:
Technical data of the wireless actuators, teach-in list, operating distances and contents of Eltako Wireless telegrams
[https://www.eltako.com/fileadmin/downloads/en/_main_catalogue/Gesamt-Katalog_ChT_gb_highRes.pdf
](https://www.eltako.com/fileadmin/downloads/en/_main_catalogue/Gesamt-Katalog_ChT_gb_highRes.pdf)
Page T-16
CONFIRMATION TELEGRAMS OF BIDIRECTIONAL ACTUATORS.

FSB61NP-230V, FSB71, FJ62/12-36V DC, FJ62NP-230V
```

ORG=         0x05
Data_byte3 = 0x70 = upper stop position,
             0x50 = lower stop position,
             0x01 = Start up,
             0x02 = Start down

If the actuator is stopped before the end of RV, only the actual elapsed time
is sent indicating the direction in a ORG7 message with the same ID! This is
also the info that the engine has stopped now.

ORG =        0x07
Data_byte3 = driving time in 100 ms MSB
Data_byte2 = driving time in 100 ms LSB
Data_byte1 = 0x01 = driven up or 0x02 = driven down
Data_byte0 = 0x0A (not blocked) 
             or 
             0x0E (blocked)

Remark: The RV time must be set on the device so that the end position is
always reached. If the roller shutter is already at an end position, the relay is
switched on receipt of a drive command anyway (0x01 or 0x02 is sent) and it is
switched off on expiry of the RV. (0x70 or 0x50 is sent).

```

See also:
[https://github.com/openhab/openhab-addons/pull/16909#issuecomment-2197453269](https://github.com/openhab/openhab-addons/pull/16909#issuecomment-2197453269)
There ist a description of a trace of the status telegramms of an Eltako FJ62 rollershutter.

Reply:
Gentle ping @fruggy83